src/pages/admin/postings/edit.vue

```diff
@@ -98,16 +101,13 @@ const onDelete = async () => {
 
 <template>
   <div class="w-full max-w-9xl mx-auto">
-    <!-- Page header -->
```

Comment:
UNDO ALL THE CHANGES NOT RELATED TO THE TASK, CREATE A SEPARATE PR FOR THIS IF YOU WANT